src/common/components/img-gallery/ImgGallery.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import style from 'common/components/scss/ImgGallery.module.scss'
+
+import noImage from 'assets/images/no-image.png'
+
+const ImgGallery = () => (
+  <div className={style.container}>
+    <div className={`${style['container__content']}`}>
+      <img src={noImage} alt="" />
+    </div>
```

Comment:
Create separate component for this element and display it by map function

Reply:
**className={style.container__content}**